lib/components/molecules/CookieDisclaimer/CookieDisclaimer.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import Button from '../../atoms/Button';
+import Icon from '../../atoms/Icon';
+import type { cookieDisclaimerProps } from './types';
+import styles from './CookieDisclaimer.style';
+
+let [isVisible, setIsVisible] = [true, null];
+
+export const toggleIsVisibleState = () => {
+  setIsVisible(!isVisible);
+};
+
+export const consentAccepted = () => {
+  // TO-DO: cookie Drop via some custom hook needs to be implemented.
+  toggleIsVisibleState();
+};
+
+const CookieDisclaimer = ({
+  acceptBtn,
+  rejectBtn,
+  className,
+  children,
+  showDisclaimer,
+  contentWRTCTAButtons,
+  marginBtnCls,
+}: cookieDisclaimerProps) => {
+  const { type, text } = acceptBtn;
+  const { rbType, rbText } = rejectBtn;
+  [isVisible, setIsVisible] = useState(showDisclaimer);
+
+  const disclaimerElement = isVisible ? (
+    <div className={className}>
+      <Button tertiary onClick={toggleIsVisibleState}>
+        <Icon id="close" />
+      </Button>
+      <div className={contentWRTCTAButtons}>
```

Comment:
Very difficult to understand variable name --> contentWRTCTAButtons

Reply:
Updated.